examples/redis-cluster-gke/README.md

```diff
@@ -0,0 +1,55 @@
+# Deploying Redis Cluster on GKE
+
```

Comment:
Add a prerequisites section:
1. Gcloud SDK, kubectl, git etc
2. Access to a GCP project and using an id with appropriate privileges.

Reply:
Added pre-requisites.